src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import StudioApiService from '../data/services/StudioApiService';
+
+var selectedForumTool; // eslint-disable-line no-unused-vars
+var setSelectedFunctions = [];
+
+function resetSelected() {
+  for (var i = 0; i < setSelectedFunctions.length; i++) {
+    setSelectedFunctions[i](false);
+  }
+}
+
+function ForumCard(props) {
+  const [hovered, setHovered] = useState(false);
+  const toggleHover = () => setHovered(!hovered);
+  const [selected, setSelected] = useState(false);
+  setSelectedFunctions.push(setSelected);
+  const toggleSelected = () => {
+    if (selected === false) {
+      if (props.locked) return;
+      resetSelected();
+      setSelected(true);
+      selectedForumTool = props.id;
+    } else {
+      setSelected(false);
+      selectedForumTool = null;
+    }
+  }
+  return (
+    <div className={hovered || selected ? 'col col-4 card forum-card-highlight' : 'col col-4 card'}>
+      <div className="card-body"
+        onMouseEnter={toggleHover}
+        onMouseLeave={toggleHover}
+        onMouseUp={toggleSelected}
+      >
+        {props.locked && (
+          <i className="fas fas-lock" aria-hidden="true"></i>
+        )}
+        {!props.locked && (
+          <input type="checkbox" name="checkbox" readOnly={true} className="forum-tool-checkbox form-check-input is-invalid-nodanger" checked={selected} />
+        )}
+        <div className="card-title h5"><img className="forum-tool-logo" src={props.logo} /></div>
+        <p className="card-text">{props.text}</p>
+        <p className="card-text tool-support">{props.support}</p>
+      </div>
+    </div>
+  );
+}
+
+function renderCards(forums) {
```

Comment:
Components should be start with a capital letter. Also this is not a good name. It could be called `ForumOptionsList` or `ForumCardList`. 

```suggestion
function ForumOptionsList({forums}) {
```